requirements/source/requirements-linting.in

```diff
@@ -0,0 +1,3 @@
+-r requirements-base.in
```

Comment:
Linting should't need to inherit from base.
This is a problem with the Case API, I will fix that.

Reply:
Updated